packages/material-ui-utils/src/useResizeObserver.js

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import ResizeObserverPolyfill from 'resize-observer-polyfill';
```

Comment:
I think that it would be better to degrade the UX for end-users that are not up-to-date => no responsiveness. The API is not supported by older versions of Safari and IE 11: https://caniuse.com/resizeobserver. It's marginal. I personally don't think that we need to add 2.5 kB gzipped for them. I imagine that most new projects won't care.
```suggestion

```